tests/networkpolicy_test.go

```diff
@@ -181,16 +181,22 @@ func createVMICirros(virtClient kubecli.KubevirtClient, namespace string, labels
 	return vmi
 }
 
-func checkPingSuccess(fromVmi *v1.VirtualMachineInstance, toIp string) {
-	Expect(tests.PingFromVMConsole(fromVmi, toIp, "-w 8")).To(Succeed())
+func checkPingSuccess(fromVmi, toVmi *v1.VirtualMachineInstance) {
+	ExpectWithOffset(1, toVmi.Status.Interfaces[0].IPs).NotTo(BeEmpty())
```

Comment:
Do we need to check `.Interfaces).To(HaveLen(1))` ?

Reply:
i dont think so, because we have WaitUntilReady and if there isnt interface there will be a panic and the test would fail